designer/pydesigner.py

```diff
@@ -200,6 +200,11 @@
                     help='Print out all output. This is a very messy '
                     'option. We recommend piping output to a text file '
                     'if you use this option.')
+parser.add_argument('--adv', action='store_true',
+                    help='Disables safety checks for advanced users who '
+                         'want to force a preprocessing step. Warning: '
+                         'Disabling these checks could yield '
+                         'undersirable results.')
```

Comment:
Could you put in some sort of all-caps warning to make it clear that it's for advanced users only?

Reply:
Clarified in 20470fb8c144d076436219e63a11e47ae901646d